### ISSUE TITLE
"packer init" fails to get sha256 checksum file

### ISSUE THREAD

**Issue by zhiguangwang** [kingsoftcloud/packer-plugin-ksyun#1]
The `packer init` command fails to get sha256 checksum file, because `SHA256SUMS` is not named correctly in GitHub Releases.

## Example Code

plugins.pkr.hcl

```hcl
packer {
  required_plugins {
    ksyun = {
      version = ">= 0.0.2"
      source = "github.com/kingsoftcloud/ksyun"
    }
  }
}
```

## Error Output

```
$ packer init .
could not get sha256 checksum file for github.com/kingsoftcloud/ksyun version 0.0.2.
Is the file present on the release and correctly named ?
GET https://github.com/kingsoftcloud/packer-plugin-ksyun/releases/download/v0.0.2/packer-plugin-ksyun_v0.0.2_SHA256SUMS: 404  []
$
```

**Comment by xuyaming0800** [comment]
This is a release bug
It will be fix v0.0.3

**Comment by zhiguangwang** [comment]
Please read the following documentation:

- https://www.packer.io/docs/plugins#plugin-location

And this repo as an example:

- https://github.com/hashicorp/packer-plugin-docker/releases
